From 058d18fdca70314bee6232f427295ebd41f45f8c Mon Sep 17 00:00:00 2001
From: Andrew Savchenko <bircoph@gmail.com>
Date: Tue, 1 Jan 2019 10:25:47 +0300
Subject: [PATCH 05/14] uname

(cherry picked from commit 020c6c42bf001b53f054cafbde739a71d4f07508)
(cherry picked from commit 99d126271f3302e84252a980639f9e6f39ef8208)
---
 src/cf/sysname.m4 | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/cf/sysname.m4 b/src/cf/sysname.m4
index acc8e7113..cd78a6192 100644
--- a/src/cf/sysname.m4
+++ b/src/cf/sysname.m4
@@ -226,7 +226,7 @@ else
                         AFS_SYSNAME="ia64_linuxXX"
                         ;;
                 powerpc-*-linux*)
-                        AFS_SYSNAME="`/bin/arch`_linuxXX"
+                        AFS_SYSNAME="`uname -m`_linuxXX"
                         ;;
                 powerpc64-*-linux*)
                         AFS_SYSNAME="ppc64_linuxXX"
@@ -244,7 +244,7 @@ else
                         AFS_SYSNAME="s390x_linuxXX"
                         ;;
                 sparc-*-linux*)
-                        AFS_SYSNAME="`/bin/arch`_linuxXX"
+                        AFS_SYSNAME="`uname -m`_linuxXX"
                         ;;
                 sparc64-*-linux*)
                         AFS_SYSNAME="sparc64_linuxXX"
-- 
2.34.1

